.github/workflows/daily.yml

```diff
@@ -32,29 +31,58 @@ on:
       use_git_ref:
         description: "git branch or sha to use"
         default: "unstable"
-
+  workflow_call:
+    inputs:
+      skipjobs:
+        description: "jobs to skip (delete the ones you wanna keep, do not leave empty)"
+        required: false
+        type: string
+        default: "valgrind,sanitizer,tls,freebsd,macos,alpine,32bit,iothreads,ubuntu,rpm-distros,malloc,specific,fortify,reply-schema,arm,lttng"
+      skiptests:
+        description: "tests to skip (delete the ones you wanna keep, do not leave empty)"
+        required: false
+        type: string
+        default: "valkey,modules,sentinel,cluster,unittest,large-memory"
```

Comment:
nit: I would just make these defaults empty. The dispatch has them all listed since you need to manually delete the ones you want to run, which is easier than listing everything else. It seems like by default we want to run everything.